pages/components/toast/toast.css

```diff
@@ -0,0 +1,40 @@
+@import "../../../styles/typography.css";
+@import "../../../styles/variable.css";
+@import "../../../styles/reset.css";
+@import "../../../styles/header.css";
+@import "../../../styles/drawer.css";
+@import "../../../styles/padding.css";
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+
+.toast-wrapper {
+  background: rgb(50, 50, 50);
+  padding: 1rem;
+  border-radius: 12px;
+  width: 100%;
+  max-width: 700px;
+}
+
+.toast-text {
+  color: var(--white-color);
```

Comment:
don't use color names in the variable, if you want to change color then you have to change name also. Use generic names like text-color or toast-color

Reply:
Used for specific purpose.